### ISSUE TITLE
notcurses-info draws sixel logo too high

### ISSUE THREAD

**Issue by dankamongmen** [dankamongmen/notcurses#2055]
In kitty, the current tip-of-master `notcurses-info` draws the logo at the correct place in all instances in both Kitty and Wezterm when using kitty graphics. With Sixel, however, the logo and URL are drawn a line too high--this happens in Wezterm's default configuration, XTerm, and Alacritty.

**Comment by dankamongmen** [comment]
top is wezterm with kitty graphics. bottom is wezterm with sixel graphics.

![2021-08-13-030734_923x1417_scrot](https://user-images.githubusercontent.com/143473/129318369-6c7fb7e7-6cfa-49f5-866d-1553b527498f.png)
![2021-08-13-030738_888x1417_scrot](https://user-images.githubusercontent.com/143473/129318371-9caaed98-4317-4eb1-8260-8ab9ac6449a1.png)

**Comment by dankamongmen** [comment]
also note that with sixel, we're not coloring the last line of unicode output properly. it's grey instead of magentish.

**Comment by dankamongmen** [comment]
note that in most of these cases, we're getting sixel overhang. we resize to 3 * cellpixy and 24 * cellpixx. in this setup, that would be 75 rows and 251 columns. 75 / 6 == 12.5, so you're likely to end up with 78 rows, right?

**Comment by dankamongmen** [comment]
sure enough, sixel has a `32;55H` before its bitmap, and kitty has a `33;55H`. so why would those be computed differently based on terminal? does it possibly have something to do with sixel's last-row prohibition?

**Comment by dankamongmen** [comment]
sixel:
```
 -------------------------- notcurses debug state -----------------------------
  *************************   0x563821d22780 pile ****************************
0000 off y:   5 x:  54 geom y:   3 x:  24 curs y:   0 x:   0 0x563821d298a0 bmap
 bound 0x563821d0c890 ← 0x563821d0c900 → (nil) binds (nil)
0001 off y:   0 x:   0 geom y:  70 x:  80 curs y:   8 x:  78 0x563821d0c890 std
 bound 0x563821d0c890 ← 0x563821d22790 → (nil) binds 0x563821d298a0
 ______________________________________________________________________________
1 render, 86.69µs (86.69µs min, 86.69µs avg, 86.69µs max)
1 raster, 35.09µs (35.09µs min, 35.09µs avg, 35.09µs max)
1 write, 1.24ms (1.24ms min, 1.24ms avg, 1.24ms max)
91.96KiB (91.96KiB min, 91.96KiB avg, 91.96KiB max) 0 inputs
0 failed renders, 0 failed rasters, 0 refreshes, 0 input errors
RGB emits:elides: def 2:158 fg 2:1086 bg 1090:78
Cell emits:elides: 1248:9896 (88.80%) 98.75% 99.82% 6.68%
Bitmap emits:elides: 1:0 (0.00%) 69.88KiB (75.99%) SuM: 1 (100.00%)
[schwarzgerat](0) $
```

kitty:
```
[schwarzgerat](0) $ cat a
 -------------------------- notcurses debug state -----------------------------
  *************************   0x5590ef21da70 pile ****************************
0000 off y:   6 x:  54 geom y:   3 x:  24 curs y:   0 x:   0 0x5590ef2408a0 bmap
 bound 0x5590ef21d990 ← 0x5590ef21da00 → (nil) binds (nil)
0001 off y:   0 x:   0 geom y:  70 x: 122 curs y:   9 x:  78 0x5590ef21d990 std
 bound 0x5590ef21d990 ← 0x5590ef21da80 → (nil) binds 0x5590ef2408a0
 ______________________________________________________________________________
1 render, 136.99µs (136.99µs min, 136.99µs avg, 136.99µs max)
1 raster, 53.92µs (53.92µs min, 53.92µs avg, 53.92µs max)
1 write, 436.27µs (436.27µs min, 436.27µs avg, 436.27µs max)
63.21KiB (63.21KiB min, 63.21KiB avg, 63.21KiB max) 0 inputs
0 failed renders, 0 failed rasters, 0 refreshes, 0 input errors
RGB emits:elides: def 2:78 fg 2:1166 bg 1170:78
Cell emits:elides: 1248:15776 (92.67%) 97.50% 99.83% 6.25%
Bitmap emits:elides: 1:0 (0.00%) 39.73KiB (62.85%) SuM: 0 (0.00%)
[schwarzgerat](0) $ 
```

**Comment by dankamongmen** [comment]
we place the new plane wherever the cursor is less 3....which yeah, matches the cursor positions for the stdplane in both instances. so it's really a question of why the cursor ends up in different places.

**Comment by dankamongmen** [comment]
the previous output is the URI, which is indeed in different places.

**Comment by dankamongmen** [comment]
oh hrmmm it looks like this is coming down to whether the emoji output line hits the right border, causing the cursor to move down, since we then work relatively to that. yep, that's it. the only things with kitty graphics are kitty, which i run at 132 columns, and wezterm, which i had opened wide when i tested there. hence the correlation--in those cases, the emoji line didn't cause a linewrap. yesssssssssssssssssss.

**Comment by dankamongmen** [comment]
confirmed. with more than 80 columns, sixel looks correct.

**Comment by dankamongmen** [comment]
i think what we need here is actually a pretty deep change: when a plane is scrolling, and our x >= dimx -> x = 0 ++y. we'll need be careful about that, though. i think what happens now is exactly that, but only when we start the next output....

**Comment by dankamongmen** [comment]
resolved in 53b017432
